docs/development/plugin/development-plugin-feature-registration.asciidoc

```diff
@@ -142,7 +142,7 @@ init(server) {
   const xpackMainPlugin = server.plugins.xpack_main;
   xpackMainPlugin.registerFeature({
     id: 'dev_tools',
-    name: i18n.translate('xpack.main.featureRegistry.devToolsFeatureName', {
+    name: i18n.translate('xpack.features.devToolsFeatureName', {
```

Comment:
This file has a few references to the legacy filepath that we'll want to update:

https://github.com/elastic/kibana/blob/af66d1183ffe502b71b613e9fe7acaa2048b6013/docs/development/plugin/development-plugin-feature-registration.asciidoc#L25

https://github.com/elastic/kibana/blob/af66d1183ffe502b71b613e9fe7acaa2048b6013/docs/development/plugin/development-plugin-feature-registration.asciidoc#L48

https://github.com/elastic/kibana/blob/af66d1183ffe502b71b613e9fe7acaa2048b6013/docs/development/plugin/development-plugin-feature-registration.asciidoc#L66

Reply:
Good catch!